nifi-nar-bundles/nifi-network-bundle/nifi-network-processors/src/main/java/org/apache/nifi/processors/network/ParseNetflowv5.java

```diff
@@ -178,7 +178,7 @@ private void generateJSON(final List<FlowFile> multipleRecords, final ProcessSes
             results.set("port", mapper.valueToTree(parser.getPortNumber()));
             results.set("format", mapper.valueToTree("netflowv5"));
 
-            recordFlowFile = session.clone(flowFile);
+            recordFlowFile = session.create(flowFile);
```

Comment:
Not related to this PR, but for future reference if you choose `INPUT_ALLOWED` you'll need to do something like this: `flowFile != null ? session.create(flowFile) : session.create()` because `ProcessSession`'s impl doesn't handle a null input.